client/src/style.css

```diff
@@ -1,3 +1,13 @@
-h1, p {
-  font-family: Lato;
+/* html {
```

Comment:
Uncomment this is you want to see what it looks like with a gradient background. 

> Hint: it's not very good